data/json/items/generic.json

```diff
@@ -2587,5 +2587,29 @@
         "weight": 5000,
         "volume": 3,
         "category": "spare_parts"
+    },
+    {
+		"id": "light_bulb",
+		"type":"GENERIC",
+		"name": "light bulb",
+		"description": "A rather outdated light bulb used in all sorts of light equipment.",
+		"symbol": ",",
+		"color": "white",
+		"material": "glass",
+		"weight": 10,
+		"volume": 1,
+		"price": 500
+    },
+	{
+		"id": "car_headlight",
```

Comment:
It's better (more consistent) to put this item in file with other vehicle parts (```items\vehicle_parts.json```).